chain/chain/src/test_utils.rs

```diff
@@ -9,6 +9,7 @@ use num_rational::Rational;
 use tracing::debug;
 
 use near_chain_configs::ProtocolConfig;
+use near_chain_configs::MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA;
```

Comment:
The code style we’re using is to group use from the same module, e.g.:

```
 use near_chain_configs::{MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA, ProtocolConfig};
```

Reply:
Good catch! Changed it based on your request.